charts/connect/values.yaml

```diff
@@ -241,6 +241,12 @@ operator:
   # Denotes whether the 1Password Operator will be deployed
   create: false
 
+  # Denotes whether the 1Password Operator will use 1Password Connect to access secrets
+  useWithConnect: true
+
+  # Denotes whether the 1Password Operator will use 1Password Service Account to access secrets
+  useWithServiceAccount: false
```

Comment:
With this change, we now make it intentional for the user whether they want to use service accounts. Simply providing the variable is no longer possible.

From my understanding and checking #228 for the change here, what we were doing before is that we'd set these variables regardless on how they were provided. We can no longer do that now due to conflicting configs (service account vs Connect). Would it possible to detect even if not explicitly specified in the Helm chart? If not, then this might be the only way we can make this work.